tests/test_actions.py

```diff
@@ -1409,14 +1409,35 @@ def test_defaults(self, mocked_webdriverwait, mocked_ec, Tester) -> None:
 
         Wait.for_the(test_target).perform_as(Tester)
 
-        mocked_webdriverwait.assert_called_once_with(mocked_browser, settings.TIMEOUT)
+        mocked_webdriverwait.assert_called_once_with(
+            mocked_browser, settings.TIMEOUT, settings.POLLING
+        )
         mocked_ec.visibility_of_element_located.assert_called_once_with(test_target)
         mocked_webdriverwait(
             mocked_browser, settings.TIMEOUT
         ).until.assert_called_once_with(
             mocked_ec.visibility_of_element_located(test_target.locator)
         )
 
+    @mock.patch("screenpy_selenium.actions.wait.EC", autospec=True)
+    @mock.patch("screenpy_selenium.actions.wait.WebDriverWait", autospec=True)
+    def test_override(self, mocked_webdriverwait, mocked_ec, Tester) -> None:
+        test_target = Target.the("foo").located_by("//bar")
+        mocked_ec.visibility_of_element_located.__name__ = "foo"
```

Comment:
What is this part for?

Reply:
Pretty sure that was in case you run the tests with logging on.  The `__name__` of an object is used to create the logging in speech tools but natively mock objects dont have names?

Reply:
```
test_actions.py::TestWait::test_override 
-------------------------------- live log setup --------------------------------
16:00:03   INFO Tester enters, stage left!
-------------------------------- live log call ---------------------------------
16:00:03   INFO Tester waits up to 4 seconds using foo with [foo]
PASSED                                                                   [100%]
```
vs 
```
test_actions.py::TestWait::test_override 
-------------------------------- live log setup --------------------------------
16:00:43   INFO Tester enters the frame!
-------------------------------- live log call ---------------------------------
16:00:43   INFO Tester waits up to 4 seconds using <MagicMock name='EC.visibility_of_element_located.__name__' id='4390500176'> with [foo]
PASSED                                                                   [100%]
```